### PR TITLE
Fix wrong version help string in `pyflyte register`

### PR DESCRIPTION
**Pull request by fg91**
# TL;DR
The `click.option("-v", "--version", ..)` currently has the help string from `@click.option("--service-account", ...)`.

## Type
 - [x] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/flyteorg/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

**Review by kumare3 — DISMISSED** [review]

**Comment by kumare3** [comment]
@fg91  can you dco sign off?

**Review by kumare3 — APPROVED** [review]

**Review by pingsutw — APPROVED** [review]

**Comment by fg91** [comment]
@pingsutw could you please merge?

**Comment by pingsutw** [comment]
Merged, thank you
